test/lib/widgets/product_license_translations_test.rb

```diff
@@ -45,6 +45,7 @@
 
       before do
         allow(Yast::UI).to receive(:TextMode).and_return(true)
+        allow(Yast::Builtins).to receive(:getenv).with("TERM").and_return("iterm")
```

Comment:
we should split this context instead of modifying it:
- when running on textmode, with iterm
- when running on textmode, without iterm